use_cases/eluc/app/components/filter.py

```diff
@@ -183,8 +186,12 @@ def update_filter_sliders(results_json: dict[str: list]) -> list:
                 outputs[0].append(min_val_rounded)
                 outputs[1].append(max_val_rounded)
                 outputs[2].append([min_val_rounded, max_val_rounded])
-                outputs[3].append({min_val_rounded: f"{int(100*min_val_rounded)}",
-                                   max_val_rounded: f"{int(100*max_val_rounded)}"})
+                if outcome == "change":
+                    outputs[3].append({min_val_rounded: f"{int(100*min_val_rounded)}",
+                                       max_val_rounded: f"{int(100*max_val_rounded)}"})
+                else:
+                    outputs[3].append({min_val_rounded: f"{min_val_rounded:.2f}",
+                                       max_val_rounded: f"{max_val_rounded:.2f}"})
```

Comment:
We scale to 2 decimals if we're not looking at change, otherwise we round to the nearest percent